test/Bin/GameTest/GameTests.cpp

```diff
@@ -1629,6 +1629,15 @@ GAME_TEST(Issues, Issue1197) {
     EXPECT_EQ(deaths.delta(), 1);
 }
 
+// 1200
+
+GAME_TEST(Issues, Issue1255) {
+    // Cant buy green wand
+    EXPECT_FALSE(pParty->hasItem(ITEM_FAIRY_WAND_OF_LASHING));
```

Comment:
```cpp
auto wandTape = tapes.hasItem(ITEM_FAIRY_WAND_OF_LASHING);
...
EXPECT_EQ(wandTape, tape(false, true));
```

Otherwise, the first check happens in the main menu.